src/Microsoft.Extensions.Options.DataAnnotations/DataAnnotationValidateOptions.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+
+namespace Microsoft.Extensions.Options
+{
+    /// <summary>
+    /// Implementation of <see cref="IValidateOptions{TOptions}"/>
```

Comment:
Can maybe elaborate on this a bit more? This comment is just stating what can be seen from the type declaration anyway.